src/cmakeProject.ts

```diff
@@ -2747,9 +2748,9 @@ export class CMakeProject {
         return this.onUseCMakePresetsChangedEmitter.event;
     }
 
-    public hideBuildButton: boolean = false;
-    public hideDebugButton: boolean = false;
-    public hideLaunchButton: boolean = false;
+    public hideBuildButton: boolean = vscode.workspace.getConfiguration('cmake').get('useProjectStatusView', true) ? true : false;
```

Comment:
Can you add a helper function for these to avoid repeating `vscode.workspace.getConfiguration('cmake').get('useProjectStatusView', true) ? true : false` three times?

Reply:
This is going to stay always false. I was trying to fix a bug and forgot to change it back when I realized this wasn't the root.

Reply:
stay as in at the start (the old code)